evm_loader/tests/test_parallel.py

```diff
@@ -25,16 +28,38 @@ def test_create_same_accounts(
     treasury_pool: TreasuryPool,
 ):
     second_account = make_new_user(evm_loader)
-    deposit_neon(evm_loader, operator_keypair, user_account.eth_address, BILLION)
-    deposit_neon(evm_loader, operator_keypair, second_account.eth_address, BILLION)
+
+    create_same_accounts_internal(
```

Comment:
It is powerful to use `subtest` feature for such cases. See https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests.

Reply:
Refactored, done.